libcxx/include/__random/uniform_int_distribution.h

```diff
@@ -64,7 +64,7 @@ class __independent_bits_engine {
   _LIBCPP_HIDE_FROM_ABI __independent_bits_engine(_Engine& __e, size_t __w);
 
```

Comment:
Unrelated to this file: let's add a release note for this optimization. You can add it in `libcxx/docs/ReleaseNotes/21.rst`.

Reply:
Do you have a suggestion on how to word this exactly? Should I just put something like this under "Improvements and New Features"?

> The implementation of `std::uniform_int_distribution` has been optimized for common cases.